packages/rn-tester/README.md

```diff
@@ -13,7 +13,7 @@ yarn install
 
 ### Running on iOS
 
-If you are testing non-fabric component, modify [the fabric_enabled flag in RNTester's Podfile](https://github.com/facebook/react-native/blob/main/packages/rn-tester/Podfile#L24).
+If you are testing non-fabric component, modify [the fabric_enabled flag in RNTester's Podfile](https://github.com/facebook/react-native/blob/main/packages/rn-tester/Podfile#L38).
```

Comment:
Linking to a line number, without automation to check this, will inevitably become outdated again and doesn't seem worth the ongoing maintenance. Can we instead widen these instructions slightly? 🙂

```suggestion
If you are testing non-fabric component, search for and modify the `fabric_enabled` flag in [RNTester's Podfile](https://github.com/facebook/react-native/blob/main/packages/rn-tester/Podfile).
```

Reply:
@huntie Makes sense 👍🏻 Applied your suggestion